website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -113,66 +113,41 @@ models:
 
 <VersionBlock firstVersion="1.6">
 
-## Materialized view
+## Materialized Views
 
-The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html) and refreshes them for every subsequent `dbt run` you execute. For more information, see [Refresh Materialized Views](https://www.postgresql.org/docs/15/sql-refreshmaterializedview.html) in the Postgres docs.
-
-Materialized views support the optional configuration `on_configuration_change` with the following values: 
-- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The following index action can be applied without the need to rebuild the materialized view:
-  - Added
-  - Dropped
-  - Updated
-- `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped
-- `fail` &mdash; forces runs to fail if a change is detected in a materialized view 
-
-You can create a materialized view by editing _one_ of these files:
-- the SQL file for your model
-- the `dbt_project.yml` configuration file
+<Changelog>
 
-The following examples create a materialized view: 
+  - **v1.6.0:** Introduced support for `materialized_view`
 
-<File name='models/YOUR_MODEL_NAME.sql'>
+</Changelog>
 
-```sql
-{{
-  config(
-    materialized = 'materialized_view',
-    on_configuration_change = 'apply',
-  )
-}}
-```
+The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html).
+Indexes are the only configuration that is specific to `dbt-postgres`.
+The remaining configuration follows the general [materialized view](/docs/build/materializations#Materialized-View) configuration.
```

Comment:
Did I link to the `Materialized View` section of `materializations` correctly here?

Reply:
i think heading anchors are lower case. the upper-case version didn't bring me to the heading we want
```suggestion
The remaining configuration follows the general [materialized view](/docs/build/materializations#materialized-view) configuration.
```